unity-renderer/Assets/Scripts/MainScripts/DCL/WebInterface/Interface.cs

```diff
@@ -954,6 +954,18 @@ public class SaveAvatarPayload
             public AvatarModel avatar;
         }
 
+        public enum RendererAuthenticationType
+        {
+            GUEST = 0,
+            WALLET_CONNECT = 1
+        }
```

Comment:
I would suggest not using an enum here, instead create a string constants and send a string as payload for the message.
That way we don't need to maintain two enums for this since kernel only needs a string for the provider. In this case "wallet_connect"